readme.md

```diff
@@ -15,13 +15,16 @@ Please read the [Contributing Guidelines](./contributing.md) before contributing
   - [Hanlon's Razor](#hanlons-razor)
   - [Murphy's Law](#murphys-law)
   - [Godwin's Law](#godwins-law)
+  - [Bystander Effect](#bystander-effect)
```

Comment:
Please put this in the psychological syndromes section.

Reply:
ok.